libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
> Should we have a default logger in the SDK?

I've been wondering the same thing, as we've gone through these reviews.  Seems like, along with the component-specific loggers, it would be useful—possibly the most common case, even—to have a "default" component for logging.